src/lib/snark_work_lib/selector.ml

```diff
@@ -0,0 +1,95 @@
+(*
+   This file tracks the Work distributed by Selector, hence the name.
```

Comment:
Could you add a short description of what selector is, and remove the part of the paragraph starting from `Originally, these types`?